packages/lib-classifier/src/components/Classifier/components/SubjectViewer/components/InteractionLayer/InteractionLayerContainer.spec.js

```diff
@@ -2,49 +2,21 @@ import { shallow } from 'enzyme'
 import React from 'react'
 import InteractionLayerContainer from './InteractionLayerContainer'
 
-const activeStepTasksWithDrawing = [
-  {
-    instruction: 'Draw a point and line',
-    taskKey: 'T0',
-    tools: [
-      { type: 'point' },
-      { type: 'line' }
-    ],
-    type: 'drawing'
-  }
-]
-
-const activeStepTasksWithSingleChoice = [
-  {
-    answers: [{ label: 'yes' }, { label: 'no' }],
-    question: 'Is there a cat?',
-    required: true,
-    taskKey: 'T0',
-    type: 'single'
-  }
-]
-
 describe('Component > InteractionLayerContainer', function () {
   it('should render without crashing', function () {
-    shallow(<InteractionLayerContainer.wrappedComponent />)
+    shallow(
+      <InteractionLayerContainer.wrappedComponent
+        addToStream={() => {}}
+        storeSVG={() => {}}
+      />)
   })
 
   it('should render an InteractionLayer', function () {
-    const wrapper = shallow(<InteractionLayerContainer.wrappedComponent />)
+    const wrapper = shallow(
+      <InteractionLayerContainer.wrappedComponent
+        addToStream={() => {}}
```

Comment:
These props should be updated to reflect changes to the component.

Reply:
Also needs tests for the case where we've got existing marks to render.

Reply:
Are the suggested tests going to be updated with this PR or in later PRs?

Reply:
It should be done here, but whichever approach gets this out of the way fastest, really.